docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -1,4 +1,70 @@
-# Configure Secure Credential Store on z/Linux operating systems
+# Configure Secure Credential Store on headless Linux operating systems
+
+Perform the following configurations on Headless and z/Linux operating systems.
+
+## Headless Linux requirements
+
+- Ensure that you installed the Secure Credential Store requirements that are described in [System Requirements](../user-guide/systemrequirements-cli.md).
+- Unlock the Gnome keyring to allow you to load and store credentials on headless Linux operating systems. You can unlock keyrings manually or automatically.
+
+**Note:** On z/Linux operating systems, complete the steps in [Configuring z/Linux](#configuring-zlinux) before you continue.  
+
+### Unlocking keyrings manually
+
+Issue the following commands to unlock keyrings manually. You must unlock the keyrings in each user session.
```

Comment:
`to unlock keyrings` -> `to unlock the keyring`
`must unlock the keyrings` -> `must unlock the keyring`